internal/llm/agent/agent.go

```diff
@@ -38,7 +39,7 @@ func (e *AgentEvent) Response() message.Message {
 }
 
 type Service interface {
-	Run(ctx context.Context, sessionID string, content string) (<-chan AgentEvent, error)
+	Run(ctx context.Context, sessionID string, content string, attachments [][]byte) (<-chan AgentEvent, error)
```

Comment:
```suggestion
	Run(ctx context.Context, sessionID string, content string, attachments ...[]byte) (<-chan AgentEvent, error)
```

Just a small thing maybe this looks better so that you don't have to use `nil` when you don't have attachments

Reply:
yeah, you're right. will change